diff -Naur hmmer-3.0.orig/easel/devkit/autodoc hmmer-3.0/easel/devkit/autodoc
--- hmmer-3.0.orig/easel/devkit/autodoc	2014-08-27 12:20:36.951770680 +0400
+++ hmmer-3.0/easel/devkit/autodoc	2014-08-27 12:20:41.722918033 +0400
@@ -49,8 +49,8 @@
 #
 # SRE, Tue Nov 30 19:43:47 2004
 
-require  "getopts.pl";
-&Getopts('n:t');
+use Getopt::Std;
+getopts('n:t');
 $cfile = shift;
 
 if ($opt_t) { $show_api_table = 1; }
diff -Naur hmmer-3.0.orig/easel/devkit/esl-dependencies hmmer-3.0/easel/devkit/esl-dependencies
--- hmmer-3.0.orig/easel/devkit/esl-dependencies	2014-08-27 12:20:36.951770680 +0400
+++ hmmer-3.0/easel/devkit/esl-dependencies	2014-08-27 12:20:41.949925046 +0400
@@ -13,8 +13,8 @@
 # SRE, Mon Jun 11 11:15:31 2007
 # SVN $Id$
 
-require "getopts.pl"
-&Getopts('1afr');
+use Getopt::Std;
+getopts('1afr');
 
 if ($opt_1) { $show_summary_table = 1; }
 if ($opt_a) { $list_augfiles      = 1; }
diff -Naur hmmer-3.0.orig/easel/devkit/sqc hmmer-3.0/easel/devkit/sqc
--- hmmer-3.0.orig/easel/devkit/sqc	2014-08-27 12:20:36.951770680 +0400
+++ hmmer-3.0/easel/devkit/sqc	2014-08-27 12:30:32.004673781 +0400
@@ -205,12 +205,12 @@
 # SRE, Tue Aug  6 11:16:39 2002
 # SVN $Id: sqc 1796 2007-01-03 22:36:44Z eddys $
 
-require "getopts.pl";
-require "importenv.pl";
+use Getopt::Std;
+use Env;
 
 # Parse our command line
 #
-&Getopts('v');
+getopts('v');
 if ($opt_v) { $verbose = 1;           }
 
 
diff -Naur hmmer-3.0.orig/easel/testsuite/coverage_report.pl hmmer-3.0/easel/testsuite/coverage_report.pl
--- hmmer-3.0.orig/easel/testsuite/coverage_report.pl	2014-08-27 12:20:36.950770649 +0400
+++ hmmer-3.0/easel/testsuite/coverage_report.pl	2014-08-27 12:20:41.955925232 +0400
@@ -16,9 +16,9 @@
 #
 # SRE, Thu Mar  1 19:22:57 2007 (Janelia)
 # SVN $Id: coverage_report.pl 231 2008-03-25 14:43:57Z eddys $
-require  "getopts.pl";
+use Getopt::Std;
 $have_sloccount = 1;
-&Getopts('cs');
+getopts('cs');
 if ($opt_c) { $do_recompile     = 1; }
 if ($opt_s) { $have_sloccount   = 0; }
 
diff -Naur hmmer-3.0.orig/easel/testsuite/driver_report.pl hmmer-3.0/easel/testsuite/driver_report.pl
--- hmmer-3.0.orig/easel/testsuite/driver_report.pl	2014-08-27 12:20:36.950770649 +0400
+++ hmmer-3.0/easel/testsuite/driver_report.pl	2014-08-27 12:20:41.955925232 +0400
@@ -25,8 +25,8 @@
 # SRE, Fri Mar  2 10:01:44 2007 (Janelia)
 # SVN $Id: driver_report.pl 326 2009-02-28 15:49:07Z eddys $
 
-require  "getopts.pl";
-&Getopts('c');
+use Getopt::Std;
+getopts('c');
 if ($opt_c) { $do_recompile = 1; }
 
 if ($ENV{'CC'}       ne "") { $CC       = $ENV{'CC'};       } else { $CC        = "gcc"; } 
diff -Naur hmmer-3.0.orig/easel/testsuite/valgrind_report.pl hmmer-3.0/easel/testsuite/valgrind_report.pl
--- hmmer-3.0.orig/easel/testsuite/valgrind_report.pl	2014-08-27 12:20:36.950770649 +0400
+++ hmmer-3.0/easel/testsuite/valgrind_report.pl	2014-08-27 12:20:41.955925232 +0400
@@ -10,8 +10,8 @@
 #
 # SRE, Fri Mar  2 08:37:48 2007 [Janelia]
 # SVN $Id: valgrind_report.pl 231 2008-03-25 14:43:57Z eddys $
-require  "getopts.pl";
-&Getopts('c');
+use Getopt::Std;
+getopts('c');
 if ($opt_c) { $do_recompile = 1; }
 
 if ($ENV{'CC'}     ne "") { $CC     = $ENV{'CC'};     } else { $CC       = "gcc"; } 
diff -Naur hmmer-3.0.orig/profmark/rocplot.pl hmmer-3.0/profmark/rocplot.pl
--- hmmer-3.0.orig/profmark/rocplot.pl	2014-08-27 12:20:36.956770834 +0400
+++ hmmer-3.0/profmark/rocplot.pl	2014-08-27 12:20:41.955925232 +0400
@@ -2,8 +2,8 @@
 
 $nsearches = 1567;
 
-require "getopts.pl";
-&Getopts('n:X:x:');
+use Getopt::Std;
+getopts('n:X:x:');
 
 if ($opt_n) { $nsearches = $opt_n; }
 if ($opt_X) { 
